resolve.js

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var resolve = require('json-refs').resolveRefs;
+var YAML = require('js-yaml');
+var fs = require('fs');
+
+var program = require('commander');
+
+program
+  .version('2.0.0')
+  .option('-o --output-format [output]',
+          'output format. Choices are "json" and "yaml" (Default is json)',
+          'json')
+  .usage('[options] <yaml file ...>')
+  .parse(process.argv);
+
+if (program.outputFormat !== 'json' && program.outputFormat !== 'yaml') {
+  console.error(program.help());
+  process.exit(1);
+}
+
+var file = program.args[0];
```

Comment:
maybe instead of using cryptic program.args[0] let use `.arguments` specification ? 
`program.arguments('<sourceFileName>').parse(process.argv)` 
and then use `program.sourceFileName` to access the value
https://github.com/tj/commander.js/#specify-the-argument-syntax

Reply:
Consider using Commander to handle CLI --> example in central-ledger -> https://github.com/mojaloop/central-ledger/blob/master/src/handlers/index.js

Reply:
resolve.js will be removed.